.github/workflows/ci.yaml

```diff
@@ -17,9 +17,9 @@ permissions:
   contents: read
 
 jobs:
-  test:
+  lib_test:
     runs-on: ubuntu-latest
-    name: Test on OTP ${{ matrix.otp }} / Elixir ${{ matrix.elixir }}
+    name: Lib Test on OTP ${{ matrix.otp }} / Elixir ${{ matrix.elixir }}
```

Comment:
Can you make it plural? I see that I used singular initially, but after splitting it it seems that it's better to use singular:
lib_test -> lib_tests, Lib Test -> Lib Tests, feature_test -> feature_tests, etc.